docs/roadmap/DSP_roadmap.md

```diff
@@ -0,0 +1,45 @@
+# Dataspace Protocol Specification  - Scope, Roadmap and Milestones
+
+## Scope
+
+The Dataspace Protocol is a set of specifications designed to facilitate interoperable data sharing between entities governed by usage control and based on Web technologies. These specifications define the schemas and protocols required for entities to publish data, negotiate Agreements, and access data as part of a federation of technical systems termed a Dataspace.
+The web rendering of the Dataspace Protocol represents the current state on this repo's main branch: https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/ 
+
+The Dataspace Protocol aims a release as Eclipse Specification combining this specification with the Dataspace TCK (https://projects.eclipse.org/projects/technology.dataspacetck ) and at least one conformant implementation. The Eclipse Dataspace Components EDC (https://projects.eclipse.org/projects/technology.edc ) are planned as compliant implementation, others are welcome. The Dataspace Protocol targets a PAS transposition to ISO during 2025.
+
+## Status (February 2025)
+
+The Dataspace Protocol project is approved by the Eclipse Dataspace Working Group (EDWG) and the Eclipse Foundation. International Data Spaces Association IDSA contributed the specification document as the initial contribution in summer 2024. The first progress review is scheduled for Q1/2025 and the release 1 is scheduled for summer 2025. 
+
+The Dataspace Protocol has a healthy number of committers (4 organizations and 7 individuals) and contributors, see https://projects.eclipse.org/projects/technology.dataspace-protocol-base/who . 72 commits have been provided to the main branch since the project start. 
+
+The Dataspace Protocol is adopted by various industrial users. 
+
+## Resources
```

Comment:
Sounds like information that is visible within the repositories. Why do we need this here?